net/dnscrypt-proxy/files/dnscrypt-proxy.init

```diff
@@ -33,7 +33,7 @@ create_config_file() {
     config_get      client_key      $1 'client_key'     ''
     config_get      syslog_prefix   $1 'syslog_prefix'  'dnscrypt-proxy'
     config_get      query_log_file  $1 'query_log_file' ''
-    config_get      log_level       $1 'log_level'	''
+    config_get      log_level       $1 'log_level'	'5'
```

Comment:
That's very stange... because default hardcoded log level is LOG_INFO : https://github.com/jedisct1/dnscrypt-proxy/blob/803adff4388b94847c7e9e15405e0fbe6c9dab3e/src/proxy/app.c#L109

which is 6 : https://github.com/jedisct1/dnscrypt-proxy/blob/f388c7213e79819b73a87631a5fddfbe1fd666cc/src/proxy/logger.h#L33

Still confused. I don't see where does the 5 come from, even in the OpenWRT / LEDE package.